packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -100,11 +100,14 @@ export interface TableProps extends CommonProps {
   onColumnsReordered?: (e?: CustomEvent<{ columnsNewOrder: string[]; column: unknown }>) => void;
   onLoadMore?: (e?: { detail: { rowCount: number } }) => void;
   /**
-   * additional options which will be passed to [react-table´s useTable hook](https://github.com/tannerlinsley/react-table/blob/master/docs/api/useTable.md#table-options)
+   * Additional options which will be passed to [react-table´s useTable hook](https://react-table.tanstack.com/docs/api/useTable#table-options)
    */
   reactTableOptions?: object;
   tableHooks?: PluginHook<any>[];
   subRowsKey?: string;
+  /**
+   * The key must consist of a valid `rowId` like `{ 2: true }` or `{ ['0.2.0']: true }` for nested rows.
```

Comment:
I think I would omit the array braces in the comment for the nested rows example